controllers/controllers.js

```diff
@@ -117,6 +117,23 @@ exports.patchArticle = (req, res, next) => {
     })
 };
 
+//9
+exports.deleteComment = (req, res, next) => {
+  const {comment_id} = req.params;
+  console.log("controllers commetnID ::", comment_id);
+  if (isNaN(comment_id)) {
+    return res.status(400).send({ message: 'Invalid comment_id' });
+    
+}
```

Comment:
Does PSQL not throw an error for this?

Reply:
psql throws an error for this// i thought i need also to write it in controller
app.use((error, req, res, next) => {
  if (error.code === '22P02') {
    res.status(400).send({ message: 'Invalid input syntax'})
  }...

![image](https://github.com/annc4st/be-nc-news-project/assets/22368936/8201c5c3-2c0a-4f4b-bccb-91f8b57da092)

Reply:
If you have a PSQL error handler set up ready to recieve then we don't also need to send a new res.send